src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/Runners/AppRunnerExtensions.cs

```diff
@@ -124,6 +125,15 @@ public static async Task StartAsync(this AppRunner runner, TimeSpan timeout)
             await runner.StartAsync(cancellation.Token).ConfigureAwait(false);
         }
 
+        public static async Task<string> GetEnvironmentVariable(this AppRunner runner, string name, TimeSpan timeout)
+        {
+            using CancellationTokenSource cancellation = new(timeout);
+            ConfiguredTaskAwaitable<string> getVarTask = runner.WaitForEnvironmentVariable(name, cancellation.Token).ConfigureAwait(false);
+            await runner.SendCommandAsync(TestAppScenarios.Commands.PrintEnvironmentVariables).ConfigureAwait(false);
+            string result = await getVarTask;
```

Comment:
There is a second `await` in this method, so it can't be a straight conversion. I can convert the `SetCommandAsync` to a wait and return the `getVarTask` but this seems to be better as-is.